attestation-agent/attester/Cargo.toml

```diff
@@ -38,12 +40,14 @@ required-features = [ "bin" ]
 default = ["all-attesters"]
 all-attesters = ["tdx-attester", "sgx-attester", "az-snp-vtpm-attester", "az-tdx-vtpm-attester", "snp-attester", "csv-attester", "cca-attester"]
 
-tdx-attester = ["tdx-attest-rs"]
+tdx-attester = ["tsm-report", "tdx-deprecated-getquote-ioctl"]
+tdx-deprecated-getquote-ioctl = ["tdx-attest-rs"]
```

Comment:
Is getquote by ioctl already deprecated?

Reply:
Ok. I prefer a name without `deprecated`. But really it does not matter

Reply:
I'm OK to drop `deprecated`.

Reply:
dropped